python-main.js

```diff
@@ -669,7 +670,7 @@ function web_editor(config) {
                     window.open('data:application/octet;charset=utf-8,' + encodeURIComponent(output), '_newtab');
                 } else {
                     var blob = new Blob([output], {type: "text/plain"});
-                    saveAs(blob, filename);
+                    saveAs(blob, scriptName + ".py");
```

Comment:
This will rename *all* files from the filesystem, instead of just main.py.

Reply:
Added if statement to only change for main.py